rainbow.py

```diff
@@ -20,32 +23,57 @@ def __init__(self, screen=None, camera=None, drive=None, colour="any"):
         self.stream = picamera.array.PiRGBArray(camera)
         self.event = threading.Event()
         self.terminated = False
-        self.MAX_AREA = 4000  # Largest target to move towards
+        self.MAX_WIDTH = 70 # Largest target to move towards
+        self.MAX_HEIGHT = 75
         self.MIN_CONTOUR_AREA = 3
+        self.LEARNING_MIN_AREA = 30
         self._colour = colour
```

Comment:
I think its not used. I don't entirely understand the significance of the underscore

Reply:
Usually an underscore prefix is used to denote a private or internal variable or method (since python doesn't have a private/public distinction like C++ etc.

does `self.colour` need initialising? otherwise unless it is set before being tested for it will not exist and raise an `AttributeError`.

Reply:
`self.colour` is always the current target colour, so needs initialising as the first value ('red')